test/samples/assert_forall.instrumented.sol

```diff
@@ -2,9 +2,16 @@
 /// Use --disarm prior to make any changes.
 pragma solidity 0.8.7;
 
-contract AssertForall {
+/// Utility contract holding a stack counter
+contract __scribble_ReentrancyUtils {
     event AssertionFailed(string message);
 
+    event AssertionFailedData(int eventId, bytes encodingData);
```

Comment:
This event is not used in the instrumented contract. We no longer generate this event on-demand, right. Is it a current expectation?

Reply:
Yep this is intended. Simpler logic to always emit it